.github/workflows/sync_common_docs.yml

```diff
@@ -0,0 +1,17 @@
+name: Sync docs with gitbook
+
+on:
+  push:
+    branches: ['github-actions-readme']
+    paths: ['docs/snap/**', 'docs/sso/**', 'docs/common/**']
+
+jobs:
+  copy_cardano_snap_docs:
+    name: Copy Common Docs
+    uses: ./.github/workflows/copy_to_gitbook_repo.yml
+    with:
+      source_file: 'docs/common/'
```

Comment:
I guess this is the naming of the plugin parameters we use, but isn't this more like a `source_folder`?